frontend/package.json

```diff
@@ -22,7 +22,7 @@
         "bootstrap": "^5.2.3",
         "fam-admin-mgmt-api": "file:../client-code-gen/gen/admin-management-api",
         "fam-app-acsctl-api": "file:../client-code-gen/gen/app-access-control-api",
-        "primevue": "3.34.0",
+        "primevue": "3.45.0",
```

Comment:
this one need to handle separately, some primevue style override need revised for the newer version